include/boost/uuid/string_generator.hpp

```diff
@@ -122,23 +133,27 @@ struct string_generator {
         static char const*const digits_end = digits_begin + 22;
```

Comment:
Okay, we'll have to subtract one, as the digits_end needs to point at the trailing NULL, but I will make this change.